testing/api/test_csa_pmj.py

```diff
@@ -1,44 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8
-# pytest unit tests for spinalcordtoolbox.csa_pmj
-
-import sys
-import os
-import pytest
-import numpy as np
-
-from spinalcordtoolbox.utils import __sct_dir__
-sys.path.append(os.path.join(__sct_dir__, 'scripts'))
-from spinalcordtoolbox.image import Image
-from spinalcordtoolbox import csa_pmj
-from spinalcordtoolbox.utils import sct_test_path
-
-
-@pytest.fixture(scope="session")
-def centerline():
-    """ Get centerline ndarray from sct_testing_data/t2/t2_centerline-manual.nii.gz """
-    fname_ctl = sct_test_path('t2', 't2_centerline-manual.nii.gz')
-    im_ctl = Image(fname_ctl).change_orientation('RPI')
-    _, _, _, _, px, py, pz, _ = im_ctl.dim
-    # Create an array like output of get_centerline()
-    arr_ctl = np.array(np.nonzero(im_ctl.data))
-    arr_ctl = arr_ctl[::, arr_ctl[2, ].argsort()]
-    return arr_ctl, px, py, pz
-
-
-def test_distance_from_pmj(centerline):
```

Comment:
Can we modify @jcohenadad's branch (`jca/sb/3063-csa-measure-from-pmj`) so that these tests don't fail? That way we won't need a commit to remove this test, I think?

Reply:
actually because it uses the class Centerline, (not possible for the other method), these function don't exists in `jca/sb/3063-csa-measure-from-pmj` so we can't really test them

Reply:
but I'll remove them in the branch `jca/sb/3063-csa-measure-from-pmj`

Reply:
> actually because it uses the class Centerline, (not possible for the other method), these function don't exists in jca/sb/3063-csa-measure-from-pmj so we can't really test them

Oh! I must have misunderstood, sorry. I didn't realize the tests were only valid for your branch (and not Julien's branch). Thank you for the clarification.